canonical-web-and-design/maas-ui
# refactor(ui): reorganise AvailableStorageTable files

## Done

- Moved form field components into the folders of the form components. It didn't really make sense for them to sit on the same level when the fields are always going to be child components of the form.
- Drive by fix so "Edit partition" shows above "Remove partition" to be consistent with all the other action dropdowns

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the storage tab of a machine in Ready or Allocated state
- Add a partition if none exist
- Open the action dropdown for the partition and check that "Edit" shows above "Remove"
